babybuddy/static_src/js/tags_editor.js

```diff
@@ -0,0 +1,337 @@
+(function() {
+    /**
+     * Parse a string as hexadecimal number
+     */
+    function hexParse(x) {
+        return parseInt(x, 16);
+    }
+
+    /**
+     * Attempt to compute a high-contrast color from a background color.
+     * 
+     * (This probably should be researched better because this was 
+     * hand-crafted ad-hoc.)
+     */
+    function computeComplementaryColor(colorStr) {
```

Comment:
Perhaps we can just pull from this MIT licensed snippet? https://vanillajstoolkit.com/helpers/getcontrast/

Reply:
Excellent idea. At some point it might be worth it to include a npm-module for this kind of stuff, but this works well! :+1: